README.md

```diff
@@ -2,179 +2,86 @@
 
  [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0) [![GoDoc](https://godoc.org/github.com/pravega/pravega-operator?status.svg)](https://godoc.org/github.com/pravega/pravega-operator) [![Build Status](https://travis-ci.org/pravega/pravega-operator.svg?branch=master)](https://travis-ci.org/pravega/pravega-operator) [![Go Report](https://goreportcard.com/badge/github.com/pravega/pravega-operator)](https://goreportcard.com/report/github.com/pravega/pravega-operator) [![Version](https://img.shields.io/github/release/pravega/pravega-operator.svg)](https://github.com/pravega/pravega-operator/releases)
 
-### Project status: alpha
-
-The project is currently alpha. While no breaking API changes are currently planned, we reserve the right to address bugs and change the API before the project is declared stable.
-
 ## Table of Contents
 
  * [Overview](#overview)
- * [Requirements](#requirements)
- * [Quickstart](#quickstart)    
- * [Install the Operator](#install-the-operator)
- * [Deploying in Test Mode](#deploying-in-test-mode)
- * [Upgrade the Operator](#upgrade-the-operator)
- * [Install a sample Pravega Cluster](#install-a-sample-pravega-cluster)
- * [Scale a Pravega Cluster](#scale-a-pravega-cluster)
- * [Upgrade a Pravega Cluster](#upgrade-a-pravega-cluster)
- * [Uninstall the Pravega Cluster](#uninstall-the-pravega-cluster)
- * [Uninstall the Operator](#uninstall-the-operator)
- * [Manual installation](#manual-installation)
- * [Configuration](#configuration)
+    * [Features](#features)
+    * [Project status](#project-status)
+    * [Requirements](#requirements)
+ * [Quickstart](#quickstart)
+    * [Install the Operator](#install-the-operator)
```

Comment:
Lets not mix operator install/upgrade with Pravega install/upgrade.
Pravega install upgrade is an operator feature and so it can be mentioned under the "Features" section.
As for Operator install/upgrade we can have a separate section "Deploying Pravega Operator" that links to the relevant helm chart readme.